toolchain/driver/driver.cpp

```diff
@@ -195,11 +204,23 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
   CARBON_VLOG() << "*** SemanticsIR::MakeFromParseTree done ***\n";
   if (dump_mode == DumpMode::SemanticsIR) {
     consumer.Flush();
-    semantics_ir.Print(output_stream_);
+    output_stream_ << semantics_ir;
     return !has_errors;
   }
   CARBON_VLOG() << "semantics_ir: " << semantics_ir;
 
+  CARBON_VLOG() << "*** Lower::Make ***\n";
+  llvm::LLVMContext llvm_context;
+  const LowerResult lower_result = Lower::Make(llvm_context, semantics_ir);
+  has_errors |= lower_result.has_errors;
+  CARBON_VLOG() << "*** Lower::Make done ***\n";
+  if (dump_mode == DumpMode::LLVMIR) {
+    consumer.Flush();
+    output_stream_ << lower_result;
+    return !has_errors;
+  }
+  CARBON_VLOG() << "lower: " << lower_result;
```

Comment:
(totally optional) maybe:

```suggestion
  CARBON_VLOG() << "lowered LLVM IR: " << lower_result;
```

Reply:
If you look at other CARBON_VLOGs (both in this file and elsewhere), I've generally tried to use the variable name or function being called as a key part of the string, rather than writing different strings (this should have been `lower_result` rather than `lower`, now `module`).

For me, since I'm familiar with the code, this helps me more quickly figure out what variable or action is being logged without necessarily looking at the code doing the logging. A descriptive string might help understand some of the flow, but will leave going back to the logging site to figure out what's actually being logged. Conversely, a descriptive string might be more helpful for people who aren't regularly developing on the toolchain to understand what the toolchain is doing. I think this depends on what you want the priority of vlog to be.

Was there something peculiar here that you think the typical approach doesn't work well? Do you think the vlog approach should change?